Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,223 @@
+  first pulse index and then sweep number. See also
+  :ref:`db_paplot_image_sortorder`.
+
+Trace settings
+""""""""""""""
+
+- ``Enable trace plot``: Toggle the display of the :ref:`Figure Pulse Average Trace plot`.
+- ``Vertical scale bar``: Size of the vertical scale bar in y-axis units
+
+Deconvolution
+"""""""""""""
+
+- ``Deconvolution``: Toggle the deconvolution pulse display, this is caculated
+  from the averaged pulses [#1]_. For the image plot, the average is the second
+  bottom-most row.
+- ``Tau [ms]``: Time constant [#2]_
+- ``Smoothing``: Smoothing parameter, use ``1`` to disable smoothing
+- ``Display [ms]``: Time range of the average pulse used for the deconvolution, use ``inf`` to use the full range
+
+General
+"""""""
+
+- ``Zero pulse``: Toggle Pulse Zeroing. This is done by Differentiation and Integration.
+- ``Time alignment``: Toggle time alignment of pulses from one column. See :ref:`db_paplot_timealignment` for an in-depth explanation.
+- ``Show average``: Toggle average pulse display. For the image plot, the average is the bottom-most row.
+- ``Use multiple graphs``: Toggle the display mode. When unchecked a single
+  graph is used for all traces and images, when checked a separate graph is
+  used for each PA set.
+- ``Show individual pulses``: Toggles the display of the single pulses. Turning
+  that off can increase the display update performance. The average and
+  deconvolution are still shown if enabled.
+
+Pulses
+""""""
+
+- ``Starting pulse``: First pulse index to display, 0-based.
+- ``Ending pulse``: Last pulse index to display, use ``inf`` to use the last pulse.
+- ``Fixed pulse length``: Choose the calculation method of the pulse lengths.
+  When unchecked the pulse length is calculated as the average pulse length
+  from the extracted pulses. When checked the ``Override length`` is used.
+- ``Override length``: Pulse length to use when no length can be calculated
+  [#3]_ or when ``Fixed pulse length`` is checked
+
+Failed Pulses
+"""""""""""""
+
+Pulses which don't cross the given level can be highlighted or hidden depending on the settings.
+
+- ``Search failed pulses``: Toggle the failed pulse search
+- ``Hide failed pulses``: When a pulse failed, hide instead of highlight.
+- ``Level``: Level in y-data units to search for failed pulses. Every pulse not
+  reaching that level is considered failing. As mentioned in
+  :ref:`db_paplot_operation_order` that search is done before zeroing.
+
+.. [#1] See `doi:10.1152/jn.00942.2007 <https://www.physiology.org/doi/full/10.1152/jn.00942.2007>`__ for the implemented method
+.. [#2] The following equation holds:
+.. math:: \tau \cdot \frac{dV}{dt} + V = f
+.. [#3] The reason that the pulse length can not be calculated can either be due to the fact that only one pulse could be found, or with pulse distribution methods like ``Poisson distribution`` from the ``Pulse Train`` stimulus epoch type.
```

Comment:
Pulse length cannot be calculated when a sweep contains a single pulse or when the pulse length is variable. e.g., "Poisson.............